src/modules/vulnerability/components/card/ReportServiceCard.vue

```diff
@@ -1,124 +1,150 @@
 <template>
-  <q-card flat class="bg-grey-2 q-pa-sm" style="border-radius: 1em">
-    <q-card-section horizontal class="justify-between">
-      <q-card-section class="col-6 flex column">
-        <div class="title">{{ vulnerabilityGroup.name }}</div>
-        <div class="flex-1 flex column">
-          <template v-if="category === ReportVulnerabilityGroupCategory.OS">
+  <div style="height: 85vh; overflow: auto">
+    <h5 class="q-ma-none q-mb-sm text-weight-medium">Operating System</h5>
+    <q-card flat class="bg-grey-2 q-pa-sm" style="border-radius: 1em">
+      <q-card-section horizontal class="justify-between">
+        <q-card-section class="col-6 flex column">
+          <div class="title">{{ assets.operating_system?.name || '' }}</div>
+          <div class="flex-1 flex column">
             <div class="row q-pl-lg q-mt-sm">
               <div class="col-3 text-weight-medium">
                 {{ $t('report.vulnerabilityCategory.version') }}
               </div>
-              <div class="col">{{ vulnerabilityGroup.version }}</div>
+              <div class="col">{{ assets.operating_system?.version }}</div>
             </div>
             <div class="row q-pl-lg flex-1 items-center">
               <div class="col-3 text-weight-medium">
                 {{ $t('report.vulnerabilityCategory.precision') }}
               </div>
-              <div class="col">{{ vulnerabilityGroup.precision }}</div>
+              <div class="col">{{ assets.operating_system?.accuracy }}</div>
```

Comment:
Accuracy refers to the precision with which this Operating System was guessed. 

It should be formatted with a "%" suffix:

E.g.: "94%"

<img width="948" height="545" alt="Image" src="https://github.com/user-attachments/assets/f249f4f9-740f-4509-abf9-e2e495b0d82a" />
```suggestion
              <div class="col">{{ assets.operating_system?.accuracy }}%</div>
```